modules/userId/index.js

```diff
@@ -106,6 +106,7 @@ import CONSTANTS from '../../src/constants.json';
 import {module} from '../../src/hook';
 import {unifiedIdSubmodule} from './unifiedIdSystem.js';
 import {pubCommonIdSubmodule} from './pubCommonIdSystem.js';
+import {britepoolIdSubmodule} from './britepoolIdSystem.js';
```

Comment:
Please note in PreBid 3.0 unifiedId, and pubCommonId will be separate modules not installed by default with user ID module itself.  Please remove the default installation.

Reply:
Hi. I've moved our module out of the default installation.